controller/include/pid.h

```diff
@@ -16,12 +16,13 @@ limitations under the License.
 #define PID_H
 
 #include <PID_v1.h>
-#include <stdint.h>
 
-#define BLOWERSPD_PIN 3
+#include "hal.h"
+
+constexpr AnalogPinId DPSENSOR_PIN = AnalogPinId::HAL_A0;
+constexpr PwmPinId BLOWERSPD_PIN = PwmPinId::PWM_3;
```

Comment:
I don't think this does what you want.  You want a C++17 "inline variable".  If you can get avr-gcc to compile in C++17 mode, then go for it  (I believe it's `inline constexpr foo`).  I spent about five minutes trying and it was still appending `-std=c++11`, and I'm not sure that the avr-gcc that comes with platformio is new enough (didn't check).

Otherwise I think you have to declare variables out of line and `extern` them in headers.  Which I think shouldn't be slower, because we're compiling with LTO.

Reply:
Changed to inline constexpr.